### DISCUSSION TITLE
"Join" block is making @durable ambigious

### DISCUSSION THREAD

**Discussion by eugene-mednefits** [postmanlabs/postman-flows#401] [Defect]
I am using a "join" block to join a list of ids. My flow is ... > Create Durable > Send Request > Select list > Loop Over Loop > Select id > End loop > Join > Send Request (here is where the durable became ambigious). The flow works but the error is deceiving. I hope this is enough information to help.

Sorry for the lack of screenshots.

**Comment by samay-v** [comment]
I doubt if its the join that's making the durables ambiguous, My guess is something in the loop making it act weird. Can you get a snapshot and upload it here? 
<img width="799" alt="image" src="https://user-images.githubusercontent.com/28357106/194220953-debda397-4d57-49fa-bd69-b827461cc3c7.png">

**Reply by eugene-mednefits** [reply]
Here is a snapshot of part of the flow
![image](https://user-images.githubusercontent.com/77720455/195255743-11c0e70a-06ee-4dd0-9649-7350809dda12.png)

**Comment by balthatrix** [comment]
I'm having the same issue. 

I'm generating some data, and trying to construct an email. After the join block, the durables from the initial part of the flor (where I define 'partner_member_id') is not recognized.
![Screenshot 2022-12-31 at 12 23 14 PM](https://user-images.githubusercontent.com/5818492/210152518-8d3621c7-5fca-4920-9799-e1471195f882.png)
![Screenshot 2022-12-31 at 12 22 33 PM](https://user-images.githubusercontent.com/5818492/210152519-565a9ca2-7f90-4a16-8d94-e82e4f577641.png)

**Reply by viko42** [reply]
You seems not up-to-date @balthatrix, upgrade and use lastest blocks
